README.md

```diff
@@ -22,7 +22,7 @@ See the section about [running tests](https://facebook.github.io/create-react-ap
 ### `npm run build`
 
 Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+It correctly bundles React in production mode and optimizes the build for the best performance..
 
 The build is minified and the filenames include the hashes.\
 Your app is ready to be deployed!
```

Comment:
- [ ] **Required:** Kindly update your README with the required section specified in the [Microverse README template.](https://github.com/microverseinc/readme-template)